docs/using.rst

```diff
@@ -402,14 +402,14 @@ to, `install Docker`_, then issue the following command:
 
 .. code-block:: shell
 
-   sudo docker run -it --rm -p 443:443 -p 80:80 --name letsencrypt \
-               -v "/etc/letsencrypt:/etc/letsencrypt" \
-               -v "/var/lib/letsencrypt:/var/lib/letsencrypt" \
-               quay.io/letsencrypt/letsencrypt:latest auth
+   sudo docker run -it --rm -p 443:443 -p 80:80 --name certbot \
+               -v "/etc/certbot:/etc/certbot" \
+               -v "/var/lib/certbot:/var/lib/certbot" \
+               quay.io/certbot/certbot:latest auth
```

Comment:
All `-v` arguments are incorrect. Additionally, we should probably keep the quay.io URL as `letsencrypt/letsencrypt`, at least for the time being.